src/commitment_scheme/kzg10/key.rs

```diff
@@ -190,6 +182,7 @@ impl CommitKey {
         numerator.ruffini(*point)
     }
 
+    #[cfg(test)]
```

Comment:
If this method is used only on tests, please move it in the `mod test` section.

If multiple tests need this function, it might be an indication that we need to rethink how we're doing the tests.
In the worst case scenario we can isolate all those functions  in a module instead of mixing them with the actual code.